en/advanced_config/esc_calibration.md

```diff
@@ -1,16 +1,21 @@
 # ESC Calibration
 
-Electronic Speed Controllers (ESC) regulate the speed (and direction) of motors based on a PWM input from the flight controller (FC). 
+> **Note** These instructions are only relevant to ESCs that use a PWM input.
 
-All ESCs in a multicopter must be configured to respond to flight controller input in the same way (across the whole input range). 
-High-quality controllers come with a factory calibration and can be just configured using the [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) and [PWM_MAX](../advanced_config/parameter_reference.md#PWM_MAX) parameters provided in the ESC technical specification.
+Electronic Speed Controllers (ESCs) regulate motor speed (and direction) based on the PWM input value from the flight controller (FC).
+The range of inputs to which an ESC will respond is configurable, and the default range can differ even between ESCs of the same model.
 
-Many low-cost models require calibration. This can/should be performed if you are unsure whether it is required!
+This calibration updates all the ESCs with the maximum and minimum PWM input values that will be supplied by the flight controller. 

```

Comment:
Presumably the "values that will be supplied by the flight controller" ARE the PWM max and min values you have set in PX4. Do we need to state what these should be set to by default?

Reply:
I don't think we need to. Important is that you validate a correct PWM_MIN after the calibration.